content/go/concepts/math-functions/terms/dim/dim.md

```diff
@@ -0,0 +1,83 @@
+---
+Title: 'Dim()'
+
+Description: 'Calculate the Maximum of the Difference between Two Arguments'
+
+Subjects:
+
+- 'Code Foundation'
+
+- 'Computer Science'
+
+Tags:
+
+- 'Math'
+
+- 'Functions'
+
+CatalogContent:
+
+- 'learn-go'
+
+- 'paths/computer-science'
+---
+The ****dim()**** function takes these two floating-point numbers (x and y), calculates their difference (difference := x - y), and then finds the maximum between this difference and 0 using the math.Max() function. The result is a single floating-point number that represents the maximum of the difference between x and y or 0.
+
+**## Syntax**
+
+```pseudo
+func Dim(x, y float64) float64
+```
+The `Dim()` function returns a single value of type `float64`. This value represents the difference between the two arguments (x-y) and 0. If the difference is greater than 0, it is returned; else, 0 is returned.
+
+Note these special cases:
+- If you pass `NaN` in this function `Dim(a, NaN)` or `Dim(NaN, b)`, then this function will return NaN.
+- if you pass `inf` in the function `Dim(+inf,inf)` the function will return NaN.
+- If you pass `-Inf` in this function `Dim(-Inf, -Inf)`, then this function will return NaN.
+
+  
+
+**## Example**
+```go
```

Comment:
We'd like an introductory sentence before the example summarizing what the example is doing.

Reply:
@SSwiniarski Hi Have made the necessary changes .